### PR TITLE
Tickets/dm 12480

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: policy/comCamMapper.yaml [hunk 1]
```diff
@@ -26,122 +26,96 @@ exposures:
...
   bias:
-    columns: date
+    columns:
+    - date
     level: Ccd
     obsTimeName: date
-    persistable: DecoratedImageF
-    python: lsst.afw.image.DecoratedImageF
+    persistable: ExposureF
+    python: lsst.afw.image.ExposureF
```
Comment (r-owen) [inline policy/comCamMapper.yaml:63]: Are these entries needed for the cals? I would have expected obs_base to provide suitable defaults.

File: config/ingest.py [hunk 2]
```diff
@@ -30,7 +30,6 @@
     'date': 'text',
     'dateObs': 'text',
     'expTime': 'double',
-    'raft': 'text',
```
Comment (r-owen) [inline config/ingest.py:33]: I'm glad you could get rid of raft.

**Review by r-owen — APPROVED** [review]
Looks fine. I am wondering if you could simplify the cal entries, but I'm not that familiar with obs_base's settings and getting the entries right is the main thing.

File: policy/comCamMapper.yaml (hunk 1 above)
Comment (mfisherlevine) [reply policy/comCamMapper.yaml:63]: I would expect so too, but couldn't find any. I agree, all these very standard calibration types should be moved to obs_base at some point.

**Pull request merged**